lib/Tags/Tags.js

```diff
@@ -0,0 +1,142 @@
+import { get, uniqBy, sortBy, difference, cloneDeep } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Callout, Pane } from '@folio/stripes/components';
+import { stripesConnect } from '@folio/stripes/core';
+
+import { TagsForm } from '@folio/stripes/smart-components';
+
+class Tags extends React.Component {
+  static manifest = Object.freeze({
+    tags: {
+      type: 'okapi',
+      path: 'tags?limit=100',
```

Comment:
100? This seems low, but I guess you c/ped this limit from the SSC implementation, right?

Reply:
Yes